boms/bom/pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.oasp.java.dev</groupId>
-    <artifactId>oasp4j</artifactId>
+    <artifactId>oasp4j-boms</artifactId>
     <version>dev-SNAPSHOT</version>
   </parent>
   <groupId>io.oasp.java</groupId>
```

Comment:
As a suggestion I think the groupId in this case should be **io.oasp.java.boms** as  _oasp4j-minimal-bom_ also has this groupId hence will give more structure to the _oasp4j-boms_ module. If this change is accepted then following files also need to be updated:

- oasp4j-modules/pom.xml

- oasp4j-samples/pom.xml

- oasp4j-starters/pom.xml

- archetype-resources/pom.xml

Reply:
I did not change the groupId on purpose in order to not break compatibility. Changing groupId or artifactId of an already published artifact is not an option for a minor release. Feel free to file a new issue that we can plan for `3.0.0` (fred).